portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,23 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Comment section</h1>");
+    response.setContentType("application/json;");
```

Comment:
Remove the `;`. Also there might be a constant somewhere in javax so you don't have to hardcode this.
```suggestion
    response.setContentType("application/json");
```